advanced/editor-context-menu-identifiers.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Editor context menu section identifiers
+title_nav: Editor context menu section identifiers
```

Comment:
This is pretty long and it starts to truncate in the nav, so maybe drop the `section` part?

Reply:
Fixed by 1adf60be42fbcb6092f348bf59472471732bec20